tests/e2e/util.go

```diff
@@ -6138,3 +6155,29 @@ func readDataFromRawBlockVolume(ns string, podName string, devicePath string, te
 	framework.ExpectNoError(err, fmt.Sprintf("failed to read device: %q inside the pod: %q", devicePath, podName))
 	gomega.Expect(strings.Contains(output, testData)).NotTo(gomega.BeFalse())
 }
+
+// verifyClusterIdConfigMapGeneration verifies if cluster id configmap gets generated by
+// csi driver in csi namespace
+func verifyClusterIdConfigMapGeneration(client clientset.Interface, ctx context.Context,
```

Comment:
should we put this under a `wait()`

Reply:
I don't think we require wait here, because as soon as csi driver detects there is no cluster ID field in vsphere config secret, it creates the configmap containing cluster ID.